modules/gitops-release-notes-1-18-2.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-18.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-18-2_{context}"]
+= Release notes for {gitops-title} 1.18.2
+
+{gitops-title} 1.18.2 is available on {OCP} 4.14, 4.16, 4.17, 4.18, 4.19, and 4.20.
+
+[IMPORTANT]
+====
+After completing the upgrade, run the `audit-operator-roles.sh` audit script to review namespace-scoped access created by the {gitops-title} Operator. The script identifies `Role` and `RoleBinding` objects that grant cross-namespace access when the _ApplicationSets in any namespace_ feature is enabled using the Argo CD custom resource (CR) (`.spec.sourceNamespaces` and `.spec.applicationSet.sourceNamespaces`). These features are disabled by default. The script scans all namespaces and reports `Role`/`RoleBinding` combinations that meet the following conditions:
```

Comment:
Could you please include the link to the script

Reply:
Fixed.